Source/Core/DolphinQt/Settings/WiiPane.cpp

```diff
@@ -120,14 +135,21 @@ void WiiPane::CreateMisc()
   m_system_language_choice->addItem(tr("Traditional Chinese"));
   m_system_language_choice->addItem(tr("Korean"));
 
-  m_pal60_mode_checkbox->setToolTip(tr("Sets the Wii display mode to 60Hz (480i) instead of 50Hz "
-                                       "(576i) for PAL games.\nMay not work for all games."));
+  m_refresh_rate_pal_mode_50hz->setToolTip(
+      tr("Sets the Wii display mode to 50Hz (576i) instead of 60Hz "
+         "(480i).\nSome games do not support 50Hz mode."));
+  m_refresh_rate_pal_mode_60hz->setToolTip(
+      tr("Sets the Wii display mode to 60Hz (480i) instead of 50Hz "
```

Comment:
This one is good, though I'd recommend adding the word standard to this for clarity, as well as mentioning PAL once again. Like this:

>Sets the Wii display mode to 60hz (480i) instead of the standard 50hz (576i) for PAL games. Only works with PAL titles that support it.

Reply:
Hah, that was literally the first iteration I had until I chanded, reverting.

Reply:
Weird, why more "PAL". I removed the "for PAL games" on purpose because it was too much. rrrright?

Reply:
Should we mention that titles that don't support it will fall back to 50 Hz? Just so nobody thinks they should avoid PAL60 for compatibility reasons.

Reply:
Sure, but I haven't seen the fallback code myself, didn't came across it. Are you sure it fallbacks?

Reply:
The way it works on console is that you set a video mode in the System Menu, then the System Menu boots the game with whatever the current VI registers are (so the video mode you have configured), and then the game is free to react on either the VI registers or try to parse the configuration itself. So all games have to in some way handle all possible options.

Official releases that have been licensed by Nintendo are required to support 50 Hz in some form, even if it's just displaying a "This game can't run in 50 Hz, please exit the game and change the output to 60 Hz" message -- some late Wii releases do this. On the other hand, no game is required to support 60 Hz at all, it's perfectly legal for a game to ignore the 60 Hz setting and switch to 50 Hz on boot regardless of the VI or system settings.

So yes, I can guarantee it falls back to 50 Hz if 60 Hz is not supported.


As a sidenote, if your system can handle the 10 Hz difference, running in 60 will give you an equivalent or better experience in almost every game; lots of games were still poorly converted to 50 Hz even as late as the Wii's lifecycle. So I support phrasing this in a way that encourages people to set the setting to 60 Hz.